src/main/resources/assets/admin/common/js/ui/treegrid/TreeGrid.ts

```diff
@@ -2005,21 +1656,21 @@ export class TreeGrid<DATA>
         this.setActive(true);
     }
 
-    private highlightRowByNode(node: TreeNode<DATA>, immediateNotification: boolean = false, callback?: Function) {
+    private highlightRowByNode(node: TreeNode<DATA>) {
```

Comment:
Codacy found an issue: [Method 'highlightRowByNode' has a complexity of 7. Maximum allowed is 4.](https://app.codacy.com/manual/enonic/lib-admin-ui/pullRequest?prid=6422415)